01-react-hooks/hooks-example/src/useState/hook.jsx

```diff
@@ -0,0 +1,14 @@
+import React, {useState} from 'react';
+
+export const MyStatus = () => {
+  // whatever we pass to useState function will become the initial value
+  const [myStatus, setMyStatus] = useState('cool');
+  return (
+    <div>
+      <p>I am {myStatus}</p>
+      <button onClick={() => setMyStatus((myStatus === 'cool') ? 'awesome' : 'cool')} >
```

Comment:
[minor] I would maybe extract `() => setMyStatus((myStatus === 'cool') ? 'awesome' : 'cool')` into a new function:
```
const handleClick = () => setMyStatus((myStatus === 'cool') ? 'awesome' : 'cool')
```

Reply:
eslint react/jsx-no-bind :)

Reply:
need to disable eslint rule, you have to bind closure variables from useState during each render for hooks to work..

Reply:
So I hope, there will be update in this eslint rule.